docs/_langs.md

```diff
@@ -4,6 +4,6 @@
   - [:es: Español](/es/)
   - [:fr: Français](/fr-fr/)
   - [:he: עברית](/he-il/)
-  - [:brazil: Português](/pt-br/)
+  - [:br: Português](/pt-br/)
```

Comment:
```suggestion
  - [:brazil: Português](/pt-br/)
```
https://github.com/ikatyang/emoji-cheat-sheet#country-flag

Still doesn't work though, despite this PR being merged: https://github.com/docsifyjs/docsify/pull/930

Reply:
When it's set to `:brazil:` we get a 404 in the console. Moving it to the 2 letter code does not have a 404.

Reply:
That's right, it doesn't because Docsify is not interpreting `:br:` as an emoji. Even if it was, https://github.githubassets.com/images/icons/emoji/br.png 404s as well.

Once Docsify get their emoji support fixed, `:brazil:` will show up correctly.